src/KafkaSchemaRegistryApiApiClient.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient;
+
+use Buzz\Exception\RequestException;
+use Exception;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\SubjectNotFoundException;
+use Jobcloud\KafkaSchemaRegistryClient\Interfaces\HttpClientInterface;
+use Jobcloud\KafkaSchemaRegistryClient\Interfaces\KafkaSchemaRegistryApiClientInterface;
+
+class KafkaSchemaRegistryApiApiClient implements KafkaSchemaRegistryApiClientInterface
+{
+    /**
+     * @var HttpClientInterface
+     */
+    private $registryClient;
+
+    /**
+     * KafkaSchemaRegistryApi constructor.
+     * @param HttpClientInterface $schemaRegistryClient
+     */
+    public function __construct(HttpClientInterface $schemaRegistryClient)
```

Comment:
```suggestion
    public function __construct(HttpClientInterface $httpClient)
```
to be consistent, otherwise we would neet to rename it to SchemaRegistryHttpClient